src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyPageAction.java

```diff
@@ -40,11 +39,11 @@ protected ActionResult execute() throws EntityDoesNotExistException {
             boolean isArchived = Logic.isCourseArchived(course.id, account.googleId);
 
             if (!isArchived && isAllowedToMakeSession) {
-                data.courses.add(course);
+                data.getCourses().add(course);
             }
         }
         
-        CourseAttributes.sortByCreatedDate(data.courses);
+        CourseAttributes.sortByCreatedDate(data.getCourses());
```

Comment:
Why not do all the processing that `courses` needs in this method before passing it into the constructor for `InstructorFeedbackEditCopyPageData`?